Documentation/concepts/networking/masquerading.rst

```diff
@@ -52,9 +52,90 @@ Implementation Modes
 --------------------
 
 eBPF-based
-  The eBPF-based implementation is the most efficient
-  implementation. It requires Linux kernel 4.19 and can be enabled with the
-  option ``enable-bpf-masquerade: true``.
+**********
+
+The eBPF-based implementation is the most efficient
+implementation. It requires Linux kernel 4.19 and can be enabled with
+the ``global.bpfMasquerade=true`` helm option (enabled by default).
+
+The current implementation depends on :ref:`the BPF NodePort feature <kubeproxy-free>`.
+The dependency will be removed in the Cilium v1.9 release.
+
+Masquerading can take place only on those devices which run the eBPF masquerading
+program. This means that a packet sent from a pod to an outside will be masqueraded
+(to an output device IPv4 address), if the output device runs the program. If not
+specified, the program will be automatically attached to the devices selected by
+:ref:`the BPF NodePort device detection metchanism <Nodeport Devices>`.
+To manually change this, use the ``global.devices`` helm option. ``cilium status``
+can be used to determine on which devices the program is running:
+
+.. parsed-literal::
```

Comment:
This command doesn`t parse any literals that I can see, it's sufficient to either:

```suggestion
.. code:: shell-session
```

or 

```suggestion
::
```

Reply:
(Note that `shell-session` highlighting is only useful if you're actually starting command lines with `$` / `#`; I'm not sure we've consistently applied a ruling on how to define these across all of the examples by this point..)